### PR TITLE
Change the colour palette for L1TStage2EMTF, add one more palette for L1TStage2CaloLayer1 and change histogram names for muon comparison in online DQM

### PR DESCRIPTION
**Pull request by pgianneios**
This PR applies changes as requested in:
https://its.cern.ch/jira/browse/CMSLITDPG-167  (Change colour palette for L1TStage2EMTF, add one more palette option for L1TStage2CaloLayer1)
and 
https://its.cern.ch/jira/browse/CMSLITDPG-355 ( Change histogram names for the muon comparison in L1TStage2)

### CODE REVIEW

File: dqmgui/style/L1TStage2CaloLayer1RenderPlugin.cc [hunk 1]
```diff
@@ -49,6 +50,7 @@ class L1T2016Layer1RenderPlugin : public DQMRenderPlugin
       for(size_t i=0; i<256; ++i) {
         auto c = new TColor(rValues[i], gValues[i], bValues[i]);
```
Comment (threus) [inline dqmgui/style/L1TStage2CaloLayer1RenderPlugin.cc]: @pgianneios can you please delete c at the end of the for loop? You create "c" 256 times. It's in intialize, but still..

File: dqmgui/style/L1TStage2EMTFRenderPlugin.cc [hunk 2]
```diff
@@ -11,20 +11,41 @@
...
+      for(size_t i=0; i<256; ++i) {
+        auto c = new TColor(rValues[i], gValues[i], bValues[i]);
```
Comment (threus) [inline dqmgui/style/L1TStage2EMTFRenderPlugin.cc]: @pgianneios also here, could you please delete c after you pass the GetNumber.

File: dqmgui/style/L1TStage2RenderPlugin.cc [hunk 3]
```diff
@@ -22,9 +22,24 @@
...
+      for(size_t i=0; i<256; ++i) {
+        auto c = new TColor(rValues[i], gValues[i], bValues[i]);
```
Comment (threus) [inline dqmgui/style/L1TStage2RenderPlugin.cc]: @pgianneios here as well, please delete it after usage

**Comment by pgianneios** [comment]
@threus I added " delete c ;" in the loops as you suggested

**Comment by threus** [comment]
@pgianneios please close this PR, it's superseded by #564

**Pull request closed without merging**